index.bs

```diff
@@ -10568,6 +10552,39 @@ class MyProcessor extends AudioWorkletProcessor {
 }
 </xmp>
 
+<h6 id="audioworkletprocess-callback-parameters" callback lt="AudioWorkletProcessCallback()">
+Callback {{AudioWorkletProcessCallback}} Parameters
+</h6>
+The following describes the parameters to the {{AudioWorkletProcessCallback}} function.
+
+In general, the {{AudioWorkletProcessCallback/inputs!!argument}} and
+{{AudioWorkletProcessCallback/outputs!!argument}} arrays will be reused
+between calls so that no memory allocation is done.  However, if the
+topology changes, because, say, the number of channels in the input or the
+output changes, new arrays are reallocated.  New arrays are also
+reallocated if any part of the
+{{AudioWorkletProcessCallback/inputs!!argument}} or
+{{AudioWorkletProcessCallback/outputs!!argument}} arrays are
+transferred.
```

Comment:
This doesn't include `parameters` object. We can (and Chrome does) freeze the object and reallocate if the topology changes.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze
https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/modules/webaudio/audio_worklet_processor.cc;l=460

Reply:
IIRC, the comments in #1933 says we'd have to do much more work than just saying "Freeze" because there's no WebIDL concept of freeze other than FrozenArrays.

Reply:
Correct. It's less ideal, but we can directly refer ECMAScript algorithm:
https://tc39.es/ecma262/#sec-object.freeze

Or start from WebIDL's FrozenArray's algorithm:
https://heycam.github.io/webidl/#es-frozen-array